geode-core/src/main/java/org/apache/geode/internal/cache/wan/AbstractGatewaySenderEventProcessor.java

```diff
@@ -503,7 +503,10 @@ protected void processQueue() {
             }
           }
           if (events.isEmpty()) {
+            System.out.println("BR: empty events");
```

Comment:
I think you forgot to remove this from other ticket.

Reply:
Not sure how those made their way into these commits, I'll remove them